app/move/views/_includes/assessment.njk

```diff
@@ -1,112 +1,118 @@
 {% for assessmentCategory in assessment %}
-  <h2 class="govuk-heading-m govuk-!-margin-bottom-2 govuk-!-margin-top-7">
-    {{ t("moves::assessment.categories." + assessmentCategory.key + ".heading") }}
-  </h2>
-
-  {% for title, answers in assessmentCategory.answersByTitle %}
-    {% call appPanel({
-      attributes: {
-        id: title | kebabcase
-      },
-      tag: {
-        text: title,
-        classes: assessmentCategory.tagClass
-      }
-    }) %}
-
-      {% set items = [] %}
-      {% for answer in answers %}
-        {% set answerHtml %}
-          {% if answer.nomis_alert_description %}
-            <h4 class="govuk-!-margin-top-0 govuk-!-margin-bottom-2">
-              {{ answer.nomis_alert_description }}
-            </h4>
-          {% endif %}
-
-          {{ answer.comments if answer.comments }}
-
-          {% if answer.created_at %}
-            <div class="govuk-!-margin-top-2 govuk-!-font-size-16">
-              {{ answer.created_at | formatDateWithDay }}
-            </div>
-          {% endif %}
-        {% endset %}
+  <section class="app-!-position-relative" data-app-section="{{ assessmentCategory.key }}">
```

Comment:
Is this data attribute used for anything?

Reply:
Yep - it's how I'm targeting the links in the e2e tests